server/server/src/schema.rs

```diff
@@ -12,7 +13,8 @@ hmdb::schema! {
         metas: <Uuid, EncryptedFileMetadata>,
         sizes: <Uuid, u64>,
         google_play_ids: <String, Owner>,
-        stripe_ids: <String, Owner>
+        stripe_ids: <String, Owner>,
+        feature_flags: <FeatureFlag, bool>
```

Comment:
I don't think this is the move, I think you should have `OneKey -> FeatureFlags` where `FeatureFlags` is a struct that contains the current settings for all the feature flags. Not every ff will be a bool.

Reply:
if you do this a lot of `server/server/src/feature_flags.rs` goes away because you can do things like `unwrap_or_default` and you can specify the default values for the flags, and they only need to get set if you change them.

Reply:
I did it slightly different than what you are mentioning which I believe is cleaner. Check it out and let me know if you instead want me to do it exactly your way.